### PR TITLE
Add Lurker in the Deep (YDSK)

### PR DESCRIPTION
**Pull request by TwentyToedToad**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Lurker in the Deep
+ManaCost:3 U U U
+Types:Enchantment Creature Fish Illusion
+PT:7/7
+K:Impending:4:2 U U
+T:Mode$ ChangesZone | Destination$ Battlefield | Origin$ Any | ValidCard$ Card.Self | Execute$ TrigSeek | TriggerDescription$ Whenever CARDNAME enters or attacks, seek a nonland card.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigSeek | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters or attacks, seek a nonland card.
+SVar:TrigSeek:DB$ Seek | Type$ Card.nonLand | RememberFound$ True
+T:Mode$ SeekAll | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigConjure | TriggerDescription$ Whenever you seek one or more cards during your turn, conjure a duplicate of each of those cards into your hand, then manifest those duplicates.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt:9]: condition missing

File: forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Lurker in the Deep
+ManaCost:3 U U U
+Types:Enchantment Creature Fish Illusion
+PT:7/7
+K:Impending:4:2 U U
+T:Mode$ ChangesZone | Destination$ Battlefield | Origin$ Any | ValidCard$ Card.Self | Execute$ TrigSeek | TriggerDescription$ Whenever CARDNAME enters or attacks, seek a nonland card.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigSeek | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters or attacks, seek a nonland card.
+SVar:TrigSeek:DB$ Seek | Type$ Card.nonLand | RememberFound$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt:8]: this won't work
instead you need to extend the call to SeekAll trigger inside SeekEffect so it also passes the soughtCards

File: forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt (hunk 1 above)
Comment (TwentyToedToad) [reply forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt:9]: Ah so `PlayerTurn$ True` ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/lurker_in_the_deep.txt:9]: yes

**Comment by paulsnoops** [comment]
@tool4ever are you good with us closing this and #6371 ? @Northmoc has picked them up because these PRs have gone quiet

**Comment by tool4ever** [comment]
yea sure was just thinking the same :)

**Pull request closed without merging**